src/test/regress/expected/am_update.out

```diff
@@ -0,0 +1,45 @@
+CREATE TABLE columnar_update(i int, j int);
+INSERT INTO columnar_update VALUES (1, 10);
+INSERT INTO columnar_update VALUES (2, 20);
+INSERT INTO columnar_update VALUES (3, 30);
+-- should fail
+UPDATE columnar_update SET j = j+1 WHERE i = 2;
```

Comment:
this didn't fail, did you mean to create table using columnar?

Reply:
Good catch. I also added a few extra tests.